wallet/src/test/java/com/android/identity_credential/wallet/OpenID4VPTest.kt

```diff
@@ -56,6 +258,37 @@ class OpenID4VPTest {
         Assert.assertEquals(Pair("credentialSubject", "dataElem"), parsePathItem("\"\$.dataElem\""))
     }
 
+    @Test
+    fun timeChecks() {
```

Comment:
Tests should generally test a single thing, for clarity & simplicity. And we don't gain much by grouping these together (the common setup has already been moved to another function). Break this up into 3 different tests, and name each function for the specific thing being tested.